CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # master [(unreleased)](https://github.com/whitesmith/rubycritic/compare/v3.2.3...master)
+* [FEATURE] Add lint format similar to Golint
```

Comment:
please add your handle here: `[@nightscape][]`

and at the bottom of the file: `[@nightscape]: https://github.com/nightscape`

Reply:
Done.